.github/workflows/collaboration-service.yml

```diff
@@ -6,6 +6,9 @@ on:
   pull_request:
     branches: [ "main" ]
 
+env:
+  REDIS_LOCAL_URI:
```

Comment:
Btw why is this blank?

Reply:
I'm not sure what local url should be used in this case, tested with localhost/127.0.0.1, so if I leave it blank it'll return undefined, thus using the default settings and will work

Reply:
Also for dump.rdb file I'll try to ignore it, it's where redis will save the dataset it is using on periodically